app/routes/events/components/EventAdministrate/Allergies.tsx

```diff
@@ -43,7 +50,21 @@ const Allergies = ({ eventId, event, error, loading, registered }: Props) => {
     return <div>{error.message}</div>;
   }
 
-  const columns = [
+  const registeredAllergies = registered.filter((registration) => {
+    return registration?.user.allergies;
+  });
+
+  const data = registeredAllergies.map(
+    (registration) => registration.user.allergies
+  );
+  const txtString = data.join('\n');
+  const allergiesTXT = URL.createObjectURL(
+    new Blob([txtString], {
```

Comment:
Could you place `txtString` directly in the Blob() if it is not used anywhere else?

Reply:
Done